### ISSUE TITLE
Class 'adLDAP\adLDAP' not found

### ISSUE THREAD

**Issue by des1roer** [edvler/yii2-adldap-module#4]
using your extension get this error

![n029ji2](https://cloud.githubusercontent.com/assets/10970793/8794367/8ef18286-2f9c-11e5-8d3c-125b53a0f8bb.png)

yii2basic\models\LoginForm.php

```
public function validatePassword($attribute, $params)
{


    if (!$this->hasErrors()) {
        $user = $this->getUser();

    $authUser = \Yii::$app->ldap->authenticate($user,$this->password);
    var_dump ($authUser);

        if (!$user || !$user->validatePassword($this->password)) {
            $this->addError($attribute, 'Incorrect username or password.');
        }
    }
}
```

**Comment by fiamma06** [comment]
I have the same problem. How to fix it?

**Comment by des1roer** [comment]
\yii2basic\vendor\edvlerblog\yii2-adldap-module\src\Ldap.php

```
    use yii\base\Component; //include YII component class

    require Yii::getAlias('@vendor') .'/adldap/adldap/src/adLDAP.php';

    use adLDAP; //include the adLDAP class
```

http://des1roer.blogspot.ru/2015/07/yii-2-ad-ldap.html

**Comment by fiamma06** [comment]
It's cool, but it would be better if the @edvler fix code.. Now he has 2 pullrequests

**Comment by edvler** [comment]
Hey,

sorry I'am off for a while...

At the moment i don't have a Yii2 installtion. Is the new Version working?

**Comment by zubines** [comment]
In the new version of yii2 also, it is not working

**Comment by vitalyzhakov** [comment]
@zubines , can you provide some additional information about error?

**Comment by vitalyzhakov** [comment]
Fixed in v1.1.0
